src/ert/gui/simulation/run_dialog.py

```diff
@@ -397,11 +397,13 @@ def _on_simulation_done(self, failed: bool, msg: str) -> None:
         self.kill_button.setHidden(True)
         self.restart_button.setVisible(self._run_model.has_failed_realizations())
         self.restart_button.setEnabled(self._run_model.support_restart)
-        self.update_total_progress(1.0, msg)
         self._notifier.set_is_simulation_running(False)
         if failed:
+            self.update_total_progress(1.0, "Failed")
```

Comment:
I think we might want to refactor this back to how it used to be, i.e. rename it: `msg -> failed_msg`? And just set it explicitly in the gui (currently we send: `"Experiment completed."` if it succeeded and disregard the message if it fails: https://github.com/equinor/ert/blob/784c5d638d4794f48930a630cc1baca69de9c712/src/ert/run_models/base_run_model.py#L338

What do you think?

Reply:
I think we should resolve the bug in this PR, and then rather do the revert/refactor in another.

We should name the variable according to what it holds, be that either message or failure message.
Combining is not very elegant, since we end up in situations like this.